Source/Extensions/Blazorise.DataGrid/Configuration/PagerOptions.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Blazorise.DataGrid.Configuration
```

Comment:
Do it under the `Blazorise.DataGrid.Configuration` namespace? I think Configuration is redundant and it can be removed completely.

Reply:
I mean... maybe the word redudant isn't the best, it's well organized under that namespace in my opinion, you could argue 
 instead that you'd want to have it have a better discovery by not putting it under a specific namespace instead?

Nothing against that. I'll change it.

Reply:
Yeah, maybe redundant isn't the best. I think only three classes are under the `Blazorise.DataGrid.Configuration` namespace, and all of them have a `DataGrid` prefix, so I think it is better to have them all under the `Blazorise.DataGrid` namespace. At least they will be easier to discover.

Reply:
Right sure I'll remove the namespace. Should we  rename to `DataGridPagerOptions` then?

Reply:
DataGridPagerOptions sounds good